cmd/common.go

```diff
@@ -156,14 +156,16 @@ func syncMain(filenames []string, dry bool, parallelism, delay int, workspace st
 	s, _ := diff.NewSyncer(currentState, targetState)
 	s.StageDelaySec = delay
 	stats, errs := solver.Solve(stopChannel, s, client, parallelism, dry)
-	if errs != nil {
-		return utils.ErrArray{Errors: errs}
-	}
 	printFn := color.New(color.FgGreen, color.Bold).PrintfFunc()
 	printFn("Summary:\n")
 	printFn("  Created: %v\n", stats.CreateOps)
 	printFn("  Updated: %v\n", stats.UpdateOps)
 	printFn("  Deleted: %v\n", stats.DeleteOps)
+	if errs != nil {
+		printErr := color.New(color.FgRed, color.Bold).PrintfFunc()
+		printErr("  Errors: %v\n\n", len(errs))
+		return utils.ErrArray{Errors: errs}
```

Comment:
Printing and returning error will result in printing of errors twice.
I'd recommend returning the error and not printing the error here as returning the error does two things:
- print the error
- make the deck unix process exit with a non-zero exit code

Reply:
Sure, we don't have to print the errors in the summary. I removed it - now the existing code has just changed order.